terraform/network.tf

```diff
@@ -1,85 +1,88 @@
-resource "aws_vpc" "vpc" {
-  cidr_block           = "10.0.0.0/16"
-  enable_dns_support   = true
-  enable_dns_hostnames = true
-  tags = {
-    environment = local.namespace
-    Name        = local.namespace
-  }
+// Not working at the moment
```

Comment:
This was a while back - Is this still the same case? have we tried to make it work?

Reply:
We will work on that.